test/index.test.js

```diff
@@ -1003,6 +1055,30 @@ describe('Segment.io', function() {
             analytics.assert(res.fromDomain === 'userdata.example1.com');
             analytics.assert(!err);
           });
+
+          describe('isCrossDomainAnalyticsEnabled', function() {
+            it('should return false when crossDomainIdServers is null', function() {
+              segment.options.crossDomainIdServers = undefined;
+
+              assert.equal(segment.isCrossDomainAnalyticsEnabled(), false);
+            });
+
+            it('should return false when crossDomainIdServers is empty', function() {
+              segment.options.crossDomainIdServers = [];
+
+              assert.equal(segment.isCrossDomainAnalyticsEnabled(), false);
+            });
+
+            it('should return false when crossDomainIdServers is null', function() {
```

Comment:
this description is inconsistent with the actual test.

Reply:
updated!